src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -522,3 +556,52 @@ export async function validate8BitsTiff(tiff: Tiff): Promise<void> {
     throw new Error(`${tiff.source.url.href} is not a 8 bits TIFF`);
   }
 }
+
+function getMapSheets(bbox: BBox, gridSize: GridSize): string[] {
```

Comment:
jsdoc comment, example/unit test would be helpful for this function

Reply:
add jsdoc comment - this is being tested implicitly by the following tests, do you think more is needed?
`describe('TiffFromMisalignedTiff', () => {`